File: Android/sources/com/android/billingclient/api/zzk.java
package com.android.billingclient.api;

import android.content.Context;
import android.content.IntentFilter;

final class zzk {
    private final Context zza;
    /* access modifiers changed from: private */
    public final PurchasesUpdatedListener zzb;
    /* access modifiers changed from: private */
    public final AlternativeBillingListener zzc;
    /* access modifiers changed from: private */
    public final UserChoiceBillingListener zzd;
    /* access modifiers changed from: private */
    public final zzby zze;
    private final zzj zzf = new zzj(this, true);
    private final zzj zzg = new zzj(this, false);
    /* access modifiers changed from: private */
    public boolean zzh;

    public zzk(Context context, PurchasesUpdatedListener purchasesUpdatedListener, zzcg zzcg, AlternativeBillingListener alternativeBillingListener, UserChoiceBillingListener userChoiceBillingListener, zzby zzby) {
        this.zza = context;
        this.zzb = purchasesUpdatedListener;
        this.zzc = alternativeBillingListener;
        this.zzd = userChoiceBillingListener;
        this.zze = zzby;
    }

    public final PurchasesUpdatedListener zzd() {
        return this.zzb;
    }

    public final void zzf() {
        this.zzf.zzb(this.zza);
        this.zzg.zzb(this.zza);
    }

    public final void zzg(boolean z) {
        IntentFilter intentFilter = new IntentFilter("com.android.vending.billing.PURCHASES_UPDATED");
        IntentFilter intentFilter2 = new IntentFilter("com.android.vending.billing.LOCAL_BROADCAST_PURCHASES_UPDATED");
        intentFilter2.addAction("com.android.vending.billing.ALTERNATIVE_BILLING");
        this.zzh = z;
        this.zzg.zza(this.zza, intentFilter2);
        if (this.zzh) {
            zzdi.zza(this.zza);
        }
        this.zzf.zza(this.zza, intentFilter);
    }
}

File: IOS/v2/com/android/billingclient/api/zzk.swift
package com.iOS.billingclient.api;.swift
import iOS.content.Context;.swift
import iOS.content.IntentFilter;.swift
final class zzk {.swift
private final Context zza;.swift
/* access modifiers changed from: private */.swift
public final PurchasesUpdatedListener zzb;.swift
/* access modifiers changed from: private */.swift
public final AlternativeBillingListener zzc;.swift
/* access modifiers changed from: private */.swift
public final UserChoiceBillingListener zzd;.swift
/* access modifiers changed from: private */.swift
public final zzby zze;.swift
private final zzj zzf = new zzj(this, true);.swift
private final zzj zzg = new zzj(this, false);.swift
/* access modifiers changed from: private */.swift
public boolean zzh;.swift
public zzk(Context context, PurchasesUpdatedListener purchasesUpdatedListener, zzcg zzcg, AlternativeBillingListener alternativeBillingListener, UserChoiceBillingListener userChoiceBillingListener, zzby zzby) {.swift
this.zza = context;.swift
this.zzb = purchasesUpdatedListener;.swift
this.zzc = alternativeBillingListener;.swift
this.zzd = userChoiceBillingListener;.swift
this.zze = zzby;.swift
}.swift
public final PurchasesUpdatedListener zzd() {.swift
return this.zzb;.swift
}.swift
public final void zzf() {.swift
this.zzf.zzb(this.zza);.swift
this.zzg.zzb(this.zza);.swift
}.swift
public final void zzg(boolean z) {.swift
IntentFilter intentFilter = new IntentFilter("com.iOS.vending.billing.PURCHASES_UPDATED");.swift
IntentFilter intentFilter2 = new IntentFilter("com.iOS.vending.billing.LOCAL_BROADCAST_PURCHASES_UPDATED");.swift
intentFilter2.addAction("com.iOS.vending.billing.ALTERNATIVE_BILLING");.swift
this.zzh = z;.swift
this.zzg.zza(this.zza, intentFilter2);.swift
if (this.zzh) {.swift
zzdi.zza(this.zza);.swift
}.swift
this.zzf.zza(this.zza, intentFilter);.swift
}.swift
}.swift
